rvar/src/Data/RVar/Prim.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE RankNTypes #-}
+-- |This is an internal interface to support the 'RVar' abstraction.  It
+-- reifies the operations provided by both MonadRandom and RandomSource in a
```

Comment:
There is no longer a `MonadRandom` or `RandomSource`?

Reply:
Yes. `random-source` and it's classes `MonadRandom` and `RandomSource` have been completely replaced by `StatefulGen`. I'll fix this comment, since it should not refer to those classes anymore

Reply:
fixed